### PR TITLE
Adding Page Builder TOC

### PR DESCRIPTION
**Pull request by bdenham**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add the Page Builder TOC to the stage build of Page Builder.

### CODE REVIEW

**Comment by bdenham** [comment]
running tests

**Comment by jeff-matthews** [comment]
@bdenham, this won't work until we add the page-builder repo to the `scripts/docs-as-code.sh` script and update all the Jenkins pipelines (devdocs-test, devdocs-build-deploy-prod, multirepo-devdocs-build-deploy-staging). See the steps in our wiki or just ask @dshevtsov for help.

File: _data/toc/page-builder.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  - label: Migrate from BlueFoot
+    url: /page-builder/docs/migration/migrate-from-bluefoot.html
+    versionless: true
+
+  - label: Migrate custom content types from BlueFoot
+    url: /page-builder/docs/migration/migrate-custom-content-types-from-bluefoot.html
+    versionless: true
+
+- label: Changelog
+  url: /page-builder/docs/CHANGELOG.html
```
Comment (jeff-matthews) [inline _data/toc/page-builder.yml]:
```suggestion
  url: /page-builder/docs/release-notes.html
```

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
Tests are green! As soon as I update the production job. I'll approve and merge this PR and kickoff a prod build.

**Pull request merged**